File: sample/rust/test_multiline.rs
fn main() {
    // Single line
    println!("debug: single line");

    // Multiline println
    println!(
        "debug: multiline message with value: {}",
        compute_value()
    );

    // Complex expression
    println!(
        "Results: x={}, y={}, sum={}",
        10,
        20,
        10 + 20
    );

    // Nested function calls
    eprintln!("debug: nested call result: {}", calculate(5, multiply(2, 3)));

    // Very long multiline
    println!(
        "DEBUG: This is a very long debug message \
         that spans multiple lines and contains \
         important information: {}",
        format_data()
    );
}

fn compute_value() -> i32 {
    dbg!(42)
}

fn calculate(a: i32, b: i32) -> i32 {
    a + b
}

fn multiply(a: i32, b: i32) -> i32 {
    a * b
}

fn format_data() -> String {
    String::from("formatted")
}

File: sample/java/TestMultiline.java
public class TestMultiline {
    public static void main(String[] args) {
        // Single line
        System.out.println("debug: single line");

        // Multiline statement
        System.out.printf(
            "debug: multiline message with value: %d\n",
            computeValue()
        );

        // Complex expression
        System.out.println(
            "Results: x=" + 10 + ", y=" + 20 + ", sum=" + (10 + 20)
        );

        // Nested function calls
        System.err.printf("debug: nested call result: %d\n", calculate(5, multiply(2, 3)));

        // Very long multiline
        System.out.println(
            "DEBUG: This is a very long debug message " +
            "that spans multiple lines and contains " +
            "important information: " + formatData()
        );
    }

    static int computeValue() {
        return 42;
    }

    static int calculate(int a, int b) {
        return a + b;
    }

    static int multiply(int a, int b) {
        return a * b;
    }

    static String formatData() {
        return "formatted";
    }
}
